src/components/Visits/index.js

```diff
@@ -43,15 +46,43 @@ const Visits = () => {
     });
   };
 
+  const updateVisit = async () => {
+    const { id } = editItem;
+    const { text, patient_name, doctor_id, date } = editItem;
+    const visit = validateVisit(editItem);
+    if (visit.isValid) {
+      const visits = api.updateVisit(id, {
```

Comment:
why do I need to destructure an element, and then write all the fields separately anyway?

Reply:
Because I don't need id from that object.

Reply:
so just put it away, and why not hand it over? what will change?